kart/lfs_commands/__init__.py

```diff
@@ -0,0 +1,86 @@
+import os
+import subprocess
+import sys
+
+import click
+
+from kart.cli_util import add_help_subcommand
+from kart.lfs_util import get_hash_from_pointer_file
+from kart.rev_list_objects import rev_list_tile_pointer_files
+from kart.repo import KartRepoState
+
+EMPTY_SHA = "0" * 40
+
+
+@add_help_subcommand
+@click.group("lfs+", hidden=True)
+@click.pass_context
+def lfs_plus(ctx, **kwargs):
+    """Git-LFS commands re-implemented in Kart to allow for spatial filtering."""
```

Comment:
can/should we fallthru other `git lfs` commands to invoke `git lfs <command>`?

Reply:
We could... if we wanted to make a full suite of git-lfs commands, where some are overridden and some still have the default behaviour. However, it's a bit out of scope for right now
- the original commands are still there if you want them - just run git-lfs
- most of them will just be for diagnostic purposes - there's only one or two that we actually need to work in Kart, ie, fetching and pushing - so it's okay if they're a little bit hidden away
- I haven't looked through them to see which ones we expect to work perfectly, and which ones will be a little weird or completely broken in Kart repos due to us disabling the git-worktree in a slightly hacky way. So, until and unless I do this, not much point exposing them again from within Kart, it would just give the expectation that they can should work properly